packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -6,6 +6,7 @@
 
 import 'package:meta/meta.dart';
 import 'package:process/process.dart';
+import 'package:http/http.dart' as http;
```

Comment:
Do not use package:http, use the io network bits instead. See base/net.dart for an example